src/python/WMCore/WMSpec/StdSpecs/DataProcessing.py

```diff
@@ -17,6 +17,17 @@ class DataProcessing(StdBase):
     for this kind of WMSpec.
     """
 
+    def __init__(self):
```

Comment:
They are, but pylint cannot see it and it complains that we are accessing these members "before" their definition. I'd rather keep it not to have false alarms in jenkins.